template/src/App.scss

```diff
@@ -1,37 +1,41 @@
-.App {
+.app {
```

Comment:
@rafayet-monon How about moving this file to `template/src/assets/stylesheets` and also rename it to `app.scss`? 🤔